.travis.yml

```diff
@@ -1,11 +1,37 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "pypy"
+matrix:
+  include:
+    # Trusty Tests
```

Comment:
Nit: might be good to comment why we use Trusty vs Xenial, context around compatibility